walletnode/node/grpc/register_sense.go

```diff
@@ -145,26 +145,27 @@ func (service *registerSense) SendRegMetadata(ctx context.Context, regMetadata *
 	request := &pb.SendRegMetadataRequest{
 		BlockHash:       regMetadata.BlockHash,
 		CreatorPastelID: regMetadata.CreatorPastelID,
+		BurnTxid:        regMetadata.BurnTxID,
 	}
 
 	_, err := service.client.SendRegMetadata(ctx, request)
 	return err
 }
 
 // ProbeImage implements node.RegisterArtwork.ProbeImage()
-func (service *registerSense) ProbeImage(ctx context.Context, image *artwork.File) ([]byte, error) {
+func (service *registerSense) ProbeImage(ctx context.Context, image *artwork.File) ([]byte, bool, error) {
 	ctx = service.contextWithLogPrefix(ctx)
 	ctx = service.contextWithMDSessID(ctx)
 
 	stream, err := service.client.ProbeImage(ctx)
 	if err != nil {
-		return nil, errors.Errorf("open stream: %w", err)
+		return nil, true, errors.Errorf("open stream: %w", err)
```

Comment:
I'm concerning `true` is a little bit hacky here.

Reply:
in fact, this return is ignore if the error !=  nil.